docs/product/issues/issue-details/replay-issues/rage-clicks.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Rage Click Issues
+sidebar_order: 40
+description: "Learn about Session Replay rage click issues."
+---
+
+If you've enabled [Session Replay](/product/explore/session-replay/), you'll be able to see rage click issues on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive page element. They are a strong signal of user frustration and most likely deserve your attention.
```

Comment:
```suggestion
If you've enabled [Session Replay](https://docs.sentry.io/product/explore/session-replay/), you'll be able to see rage click issues on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive page element. They are a strong signal of user frustration and most likely deserve your attention.
```

Reply:
why add the domain here? in dev/preview envs it'll link you to prod instead of keeping the browser on the dev server

Reply:
Oh, weird, when I clicked on it before for some reason it wasn't working properly. But it seems to be working now so we can ignore this one.